### ISSUE TITLE
UE5.3 Compile Errors : error LNK2019: unresolved external symbol

### ISSUE THREAD

**Issue by ChrisSchroeder01** [AskingQuestions/Shadeup#13]
Hello, I wanted to add a Compute Shader for writing to a render target and followed the documentation:
I created a blank plugin named "ShadeUp" after this I used the following command in cli:
```
$ shadeup
 > [COMPUTE] Compute Shader
 > Render Target
```
this works perfectly fine, but after trying to build the project (from source or via live code) i am getting the following output:
```
27>  Executing up to 6 processes, one per physical core
27>  Requested 1.5 GB memory per action, 2.69 GB available: limiting max parallel actions to 1
27>------ Building 10 action(s) started ------
27>[1/10] Compile [x64] SharedPCH.Engine.Cpp20.cpp
27>[2/10] Compile [x64] Test.cpp
27>[3/10] Compile [x64] Module.ShadeUp.cpp
27>[4/10] Resource Default.rc2
27>[5/10] Link [x64] UnrealEditor-ShadeUp.dll
27>   Creating library C:\Users\chris\Documents\Unreal Projects\Test\Plugins\ShadeUp\Intermediate\Build\Win64\x64\UnrealEditor\Development\ShadeUp\UnrealEditor-ShadeUp.sup.lib and object C:\Users\chris\Documents\Unreal Projects\Test\Plugins\ShadeUp\Intermediate\Build\Win64\x64\UnrealEditor\Development\ShadeUp\UnrealEditor-ShadeUp.sup.exp
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class FName __cdecl LegacyShaderPlatformToShaderFormat(enum EShaderPlatform)" (__imp_?LegacyShaderPlatformToShaderFormat@@YA?AVFName@@W4EShaderPlatform@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) private: void __cdecl FRHIResource::Destroy(void)const " (__imp_?Destroy@FRHIResource@@AEBAXXZ) referenced in function "public: __cdecl TRefCountPtr<struct FRHIUniformBufferLayout const >::~TRefCountPtr<struct FRHIUniformBufferLayout const >(void)" (??1?$TRefCountPtr@$$CBUFRHIUniformBufferLayout@@@@QEAA@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FShaderParametersMetadata::FShaderParametersMetadata(enum FShaderParametersMetadata::EUseCase,enum EUniformBufferBindingFlags,wchar_t const *,wchar_t const *,wchar_t const *,wchar_t const *,char const *,int,unsigned int,class TArray<class FShaderParametersMetadata::FMember,class TSizedDefaultAllocator<32> > const &,bool,struct FRHIUniformBufferLayoutInitializer *,unsigned int)" (__imp_??0FShaderParametersMetadata@@QEAA@W4EUseCase@0@W4EUniformBufferBindingFlags@@PEB_W222PEBDHIAEBV?$TArray@VFMember@FShaderParametersMetadata@@V?$TSizedDefaultAllocator@$0CA@@@@@_NPEAUFRHIUniformBufferLayoutInitializer@@I@Z) referenced in function "public: static class FShaderParametersMetadata const * __cdecl FSlimeCS::FParameters::FTypeInfo::GetStructMetadata(void)" (?GetStructMetadata@FTypeInfo@FParameters@FSlimeCS@@SAPEBVFShaderParametersMetadata@@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl FShaderParametersMetadata::~FShaderParametersMetadata(void)" (__imp_??1FShaderParametersMetadata@@UEAA@XZ) referenced in function "void __cdecl `public: static class FShaderParametersMetadata const * __cdecl FSlimeCS::FParameters::FTypeInfo::GetStructMetadata(void)'::`2'::`dynamic atexit destructor for 'StaticStructMetadata''(void)" (??__FStaticStructMetadata@?1??GetStructMetadata@FTypeInfo@FParameters@FSlimeCS@@SAPEBVFShaderParametersMetadata@@XZ@YAXXZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: static class FRDGAllocator & __cdecl FRDGAllocator::Get(void)" (__imp_?Get@FRDGAllocator@@SAAEAV1@XZ) referenced in function "private: void __cdecl TArray<class FRDGPass *,class TRDGArrayAllocator<0> >::ResizeGrow(int)" (?ResizeGrow@?$TArray@PEAVFRDGPass@@V?$TRDGArrayAllocator@$0A@@@@@AEAAXH@Z)
27>  Hint on symbols that are defined and could potentially match:
27>    "__declspec(dllimport) public: static class FObjectInitializer & __cdecl FObjectInitializer::Get(void)" (__imp_?Get@FObjectInitializer@@SAAEAV1@XZ)
27>    "__declspec(dllimport) public: static class FTaskGraphInterface & __cdecl FTaskGraphInterface::Get(void)" (__imp_?Get@FTaskGraphInterface@@SAAEAV1@XZ)
27>    "__declspec(dllimport) public: static struct FThreadStatsPool & __cdecl FThreadStatsPool::Get(void)" (__imp_?Get@FThreadStatsPool@@SAAEAU1@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl UE::RenderCore::Private::OnInvalidRDGAllocatorNum(int,unsigned __int64)" (__imp_?OnInvalidRDGAllocatorNum@Private@RenderCore@UE@@YAXH_K@Z) referenced in function "private: void __cdecl TArray<class FRDGPass *,class TRDGArrayAllocator<0> >::ResizeGrow(int)" (?ResizeGrow@?$TArray@PEAVFRDGPass@@V?$TRDGArrayAllocator@$0A@@@@@AEAAXH@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FDrawCallCategoryName::FDrawCallCategoryName(void)" (__imp_??0FDrawCallCategoryName@@QEAA@XZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl FRHICommandDispatchComputeShader::Execute(class FRHICommandListBase &)" (__imp_?Execute@FRHICommandDispatchComputeShader@@QEAAXAEAVFRHICommandListBase@@@Z) referenced in function "public: virtual void __cdecl FRHICommand<struct FRHICommandDispatchComputeShader,struct FRHICommandDispatchComputeShaderString1652>::ExecuteAndDestruct(class FRHICommandListBase &,struct FRHICommandListDebugContext &)" (?ExecuteAndDestruct@?$FRHICommand@UFRHICommandDispatchComputeShader@@UFRHICommandDispatchComputeShaderString1652@@@@UEAAXAEAVFRHICommandListBase@@AEAUFRHICommandListDebugContext@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl FRHICommandSetStaticUniformBuffers::Execute(class FRHICommandListBase &)" (__imp_?Execute@FRHICommandSetStaticUniformBuffers@@QEAAXAEAVFRHICommandListBase@@@Z) referenced in function "public: virtual void __cdecl FRHICommand<struct FRHICommandSetStaticUniformBuffers,struct FRHICommandSetStaticUniformBuffersString1965>::ExecuteAndDestruct(class FRHICommandListBase &,struct FRHICommandListDebugContext &)" (?ExecuteAndDestruct@?$FRHICommand@UFRHICommandSetStaticUniformBuffers@@UFRHICommandSetStaticUniformBuffersString1965@@@@UEAAXAEAVFRHICommandListBase@@AEAUFRHICommandListDebugContext@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl FRHICommandSetShaderUnbinds<class FRHIComputeShader>::Execute(class FRHICommandListBase &)" (__imp_?Execute@?$FRHICommandSetShaderUnbinds@VFRHIComputeShader@@@@QEAAXAEAVFRHICommandListBase@@@Z) referenced in function "public: virtual void __cdecl FRHICommand<struct FRHICommandSetShaderUnbinds<class FRHIComputeShader>,struct FRHICommandSetShaderUnbindsString1441>::ExecuteAndDestruct(class FRHICommandListBase &,struct FRHICommandListDebugContext &)" (?ExecuteAndDestruct@?$FRHICommand@U?$FRHICommandSetShaderUnbinds@VFRHIComputeShader@@@@UFRHICommandSetShaderUnbindsString1441@@@@UEAAXAEAVFRHICommandListBase@@AEAUFRHICommandListDebugContext@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class FRHICommandListImmediate & __cdecl GetImmediateCommandList_ForRenderCommand(void)" (__imp_?GetImmediateCommandList_ForRenderCommand@@YAAEAVFRHICommandListImmediate@@XZ) referenced in function "public: static void __cdecl FSlimeCSInterface::Dispatch(struct FSlimeCSDispatchParams)" (?Dispatch@FSlimeCSInterface@@SAXUFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) private: class FRHIShader * __cdecl FShaderMapResource::CreateShaderOrCrash(int)" (__imp_?CreateShaderOrCrash@FShaderMapResource@@AEAAPEAVFRHIShader@@H@Z) referenced in function "public: class FRHIComputeShader * __cdecl TShaderRefBase<class FSlimeCS,class FShaderMapPointerTable>::GetComputeShader(void)const " (?GetComputeShader@?$TShaderRefBase@VFSlimeCS@@VFShaderMapPointerTable@@@@QEBAPEAVFRHIComputeShader@@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl FShaderParameterBindings::BindForLegacyShaderParameters(class FShader const *,int,class FShaderParameterMap const &,class FShaderParametersMetadata const &,bool)" (__imp_?BindForLegacyShaderParameters@FShaderParameterBindings@@QEAAXPEBVFShader@@HAEBVFShaderParameterMap@@AEBVFShaderParametersMetadata@@_N@Z) referenced in function "void __cdecl BindForLegacyShaderParameters<class FSlimeCS::FParameters>(class FShader *,int,class FShaderParameterMap const &,bool)" (??$BindForLegacyShaderParameters@VFParameters@FSlimeCS@@@@YAXPEAVFShader@@HAEBVFShaderParameterMap@@_N@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FShader::FShader(void)" (__imp_??0FShader@@QEAA@XZ) referenced in function "public: __cdecl FSlimeCS::FSlimeCS(void)" (??0FSlimeCS@@QEAA@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FShader::~FShader(void)" (__imp_??1FShader@@QEAA@XZ) referenced in function "public: __cdecl FGlobalShader::~FGlobalShader(void)" (??1FGlobalShader@@QEAA@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FShaderType::FShaderType(enum FShaderType::EShaderTypeForDynamicCast,struct FTypeLayoutDesc &,wchar_t const *,wchar_t const *,wchar_t const *,unsigned int,int,class FShader * (__cdecl*)(void),class FShader * (__cdecl*)(struct FShaderCompiledShaderInitializerType const &),bool (__cdecl*)(struct FShaderPermutationParameters const &),enum ERayTracingPayloadType (__cdecl*)(int),void (__cdecl*)(struct FShaderPermutationParameters const &,struct FShaderCompilerEnvironment &),bool (__cdecl*)(enum EShaderPlatform,class FShaderParameterMap const &,class TArray<class FString,class TSizedDefaultAllocator<32> > &),unsigned int,class FShaderParametersMetadata const *)" (__imp_??0FShaderType@@QEAA@W4EShaderTypeForDynamicCast@0@AEAUFTypeLayoutDesc@@PEB_W22IHP6APEAVFShader@@XZP6APEAV3@AEBUFShaderCompiledShaderInitializerType@@@ZP6A_NAEBUFShaderPermutationParameters@@@ZP6A?AW4ERayTracingPayloadType@@H@ZP6AX6AEAUFShaderCompilerEnvironment@@@ZP6A_NW4EShaderPlatform@@AEBVFShaderParameterMap@@AEAV?$TArray@VFString@@V?$TSizedDefaultAllocator@$0CA@@@@@@ZIPEBVFShaderParametersMetadata@@@Z) referenced in function "public: static class FGlobalShaderType & __cdecl FSlimeCS::GetStaticType(void)" (?GetStaticType@FSlimeCS@@SAAEAVFGlobalShaderType@@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl FShaderType::~FShaderType(void)" (__imp_??1FShaderType@@UEAA@XZ) referenced in function "public: virtual void * __cdecl FGlobalShaderType::`scalar deleting destructor'(unsigned int)" (??_GFGlobalShaderType@@UEAAPEAXI@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: static class TArray<class FShaderTypeRegistration const *,class TSizedDefaultAllocator<32> > & __cdecl FShaderTypeRegistration::GetInstances(void)" (__imp_?GetInstances@FShaderTypeRegistration@@SAAEAV?$TArray@PEBVFShaderTypeRegistration@@V?$TSizedDefaultAllocator@$0CA@@@@@XZ) referenced in function "void __cdecl `dynamic initializer for 'private: static class FShaderTypeRegistration FSlimeCS::ShaderTypeRegistration''(void)" (??__E?ShaderTypeRegistration@FSlimeCS@@0VFShaderTypeRegistration@@A@@YAXXZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class TShaderRefBase<class FShader,class FShaderMapPointerTable> __cdecl FGlobalShaderMap::GetShader(class FShaderType *,int)const " (__imp_?GetShader@FGlobalShaderMap@@QEBA?AV?$TShaderRefBase@VFShader@@VFShaderMapPointerTable@@@@PEAVFShaderType@@H@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: static struct FTypeLayoutDesc & __cdecl FGlobalShader::StaticGetTypeLayout(void)" (__imp_?StaticGetTypeLayout@FGlobalShader@@SAAEAUFTypeLayoutDesc@@XZ) referenced in function "public: static struct FTypeLayoutDesc & __cdecl FSlimeCS::StaticGetTypeLayout(void)" (?StaticGetTypeLayout@FSlimeCS@@SAAEAUFTypeLayoutDesc@@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FGlobalShader::FGlobalShader(struct FShaderCompiledShaderInitializerType const &)" (__imp_??0FGlobalShader@@QEAA@AEBUFShaderCompiledShaderInitializerType@@@Z) referenced in function "public: __cdecl FSlimeCS::FSlimeCS(struct FShaderCompiledShaderInitializerType const &)" (??0FSlimeCS@@QEAA@AEBUFShaderCompiledShaderInitializerType@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class FGlobalShaderMap * __cdecl GetGlobalShaderMap(enum EShaderPlatform)" (__imp_?GetGlobalShaderMap@@YAPEAVFGlobalShaderMap@@W4EShaderPlatform@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl SetComputePipelineState(class FRHIComputeCommandList &,class FRHIComputeShader *)" (__imp_?SetComputePipelineState@@YAXAEAVFRHIComputeCommandList@@PEAVFRHIComputeShader@@@Z) referenced in function "void __cdecl FComputeShaderUtils::Dispatch<class FSlimeCS>(class FRHIComputeCommandList &,class TShaderRefBase<class FSlimeCS,class FShaderMapPointerTable> const &,class FShaderParametersMetadata const *,class FSlimeCS::FParameters const &,struct UE::Math::TIntVector3<int>)" (??$Dispatch@VFSlimeCS@@@FComputeShaderUtils@@YAXAEAVFRHIComputeCommandList@@AEBV?$TShaderRefBase@VFSlimeCS@@VFShaderMapPointerTable@@@@PEBVFShaderParametersMetadata@@AEBVFParameters@FSlimeCS@@U?$TIntVector3@H@Math@UE@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class TRefCountPtr<struct IPooledRenderTarget> __cdecl CreateRenderTarget(class FRHITexture *,wchar_t const *)" (__imp_?CreateRenderTarget@@YA?AV?$TRefCountPtr@UIPooledRenderTarget@@@@PEAVFRHITexture@@PEB_W@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGEventName::FRDGEventName(wchar_t const *,...)" (__imp_??0FRDGEventName@@QEAA@PEB_WZZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: wchar_t const * __cdecl FRDGEventName::GetTCHAR(void)const " (__imp_?GetTCHAR@FRDGEventName@@QEBAPEB_WXZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGEventScopeGuard::FRDGEventScopeGuard(class FRDGBuilder &,class FRDGEventName &&,bool,enum ERDGEventScopeFlags)" (__imp_??0FRDGEventScopeGuard@@QEAA@AEAVFRDGBuilder@@$$QEAVFRDGEventName@@_NW4ERDGEventScopeFlags@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGEventScopeGuard::~FRDGEventScopeGuard(void)" (__imp_??1FRDGEventScopeGuard@@QEAA@XZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGGPUStatScopeGuard::FRDGGPUStatScopeGuard(class FRDGBuilder &,class FName const &,class FName const &,wchar_t const *,struct FDrawCallCategoryName &)" (__imp_??0FRDGGPUStatScopeGuard@@QEAA@AEAVFRDGBuilder@@AEBVFName@@1PEB_WAEAUFDrawCallCategoryName@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGGPUStatScopeGuard::~FRDGGPUStatScopeGuard(void)" (__imp_??1FRDGGPUStatScopeGuard@@QEAA@XZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class FUniformBufferStaticBindings __cdecl FRDGParameterStruct::GetStaticUniformBuffers(void)const " (__imp_?GetStaticUniformBuffers@FRDGParameterStruct@@QEBA?AVFUniformBufferStaticBindings@@XZ) referenced in function "private: virtual void __cdecl TRDGLambdaPass<class FSlimeCS::FParameters,class `public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)'::`6'::<lambda_1> >::Execute(class FRHIComputeCommandList &)" (?Execute@?$TRDGLambdaPass@VFParameters@FSlimeCS@@V<lambda_1>@?5??DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z@@@EEAAXAEAVFRHIComputeCommandList@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGPass::FRDGPass(class FRDGEventName &&,class FRDGParameterStruct,enum ERDGPassFlags)" (__imp_??0FRDGPass@@QEAA@$$QEAVFRDGEventName@@VFRDGParameterStruct@@W4ERDGPassFlags@@@Z) referenced in function "public: __cdecl TRDGLambdaPass<class FSlimeCS::FParameters,class `public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)'::`6'::<lambda_1> >::TRDGLambdaPass<class FSlimeCS::FParameters,class `public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)'::`6'::<lambda_1> >(class FRDGEventName &&,class FShaderParametersMetadata const *,class FSlimeCS::FParameters const *,enum ERDGPassFlags,class `public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)'::`6'::<lambda_1> &&)" (??0?$TRDGLambdaPass@VFParameters@FSlimeCS@@V<lambda_1>@?5??DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z@@@QEAA@$$QEAVFRDGEventName@@PEBVFShaderParametersMetadata@@PEBVFParameters@FSlimeCS@@W4ERDGPassFlags@@$$QEAV<lambda_1>@?5??DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl FRDGUserValidation::ValidateAddPass(void const *,class FShaderParametersMetadata const *,class FRDGEventName const &,enum ERDGPassFlags)" (__imp_?ValidateAddPass@FRDGUserValidation@@QEAAXPEBXPEBVFShaderParametersMetadata@@AEBVFRDGEventName@@W4ERDGPassFlags@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGBuilder::FRDGBuilder(class FRHICommandListImmediate &,class FRDGEventName,enum ERDGBuilderFlags)" (__imp_??0FRDGBuilder@@QEAA@AEAVFRHICommandListImmediate@@VFRDGEventName@@W4ERDGBuilderFlags@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl FRDGBuilder::~FRDGBuilder(void)" (__imp_??1FRDGBuilder@@QEAA@XZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class FRDGTexture * __cdecl FRDGBuilder::FindExternalTexture(class FRHITexture *)const " (__imp_?FindExternalTexture@FRDGBuilder@@QEBAPEAVFRDGTexture@@PEAVFRHITexture@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class FRDGTexture * __cdecl FRDGBuilder::RegisterExternalTexture(class TRefCountPtr<struct IPooledRenderTarget> const &,enum ERDGTextureFlags)" (__imp_?RegisterExternalTexture@FRDGBuilder@@QEAAPEAVFRDGTexture@@AEBV?$TRefCountPtr@UIPooledRenderTarget@@@@W4ERDGTextureFlags@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class FRDGTexture * __cdecl FRDGBuilder::CreateTexture(struct FRDGTextureDesc const &,wchar_t const *,enum ERDGTextureFlags)" (__imp_?CreateTexture@FRDGBuilder@@QEAAPEAVFRDGTexture@@AEBUFRDGTextureDesc@@PEB_WW4ERDGTextureFlags@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class FRDGTextureUAV * __cdecl FRDGBuilder::CreateUAV(class FRDGTextureUAVDesc const &,enum ERDGUnorderedAccessViewFlags)" (__imp_?CreateUAV@FRDGBuilder@@QEAAPEAVFRDGTextureUAV@@AEBVFRDGTextureUAVDesc@@W4ERDGUnorderedAccessViewFlags@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: void __cdecl FRDGBuilder::Execute(void)" (__imp_?Execute@FRDGBuilder@@QEAAXXZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) private: static enum ERDGPassFlags __cdecl FRDGBuilder::OverridePassFlags(wchar_t const *,enum ERDGPassFlags)" (__imp_?OverridePassFlags@FRDGBuilder@@CA?AW4ERDGPassFlags@@PEB_WW42@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) private: void __cdecl FRDGBuilder::FlushAccessModeQueue(void)" (__imp_?FlushAccessModeQueue@FRDGBuilder@@AEAAXXZ) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) private: class FRDGPass * __cdecl FRDGBuilder::SetupParameterPass(class FRDGPass *)" (__imp_?SetupParameterPass@FRDGBuilder@@AEAAPEAVFRDGPass@@PEAV2@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) private: void __cdecl FRDGBuilder::ClobberPassOutputs(class FRDGPass const *)" (__imp_?ClobberPassOutputs@FRDGBuilder@@AEAAXPEBVFRDGPass@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl ValidateShaderParameters(class TShaderRefBase<class FShader,class FShaderMapPointerTable> const &,class FShaderParametersMetadata const *,void const *)" (__imp_?ValidateShaderParameters@@YAXAEBV?$TShaderRefBase@VFShader@@VFShaderMapPointerTable@@@@PEBVFShaderParametersMetadata@@PEBX@Z) referenced in function "void __cdecl FComputeShaderUtils::Dispatch<class FSlimeCS>(class FRHIComputeCommandList &,class TShaderRefBase<class FSlimeCS,class FShaderMapPointerTable> const &,class FShaderParametersMetadata const *,class FSlimeCS::FParameters const &,struct UE::Math::TIntVector3<int>)" (??$Dispatch@VFSlimeCS@@@FComputeShaderUtils@@YAXAEAVFRHIComputeCommandList@@AEBV?$TShaderRefBase@VFSlimeCS@@VFShaderMapPointerTable@@@@PEBVFShaderParametersMetadata@@AEBVFParameters@FSlimeCS@@U?$TIntVector3@H@Math@UE@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl SetShaderParameters(class FRHIComputeCommandList &,class FRHIComputeShader *,class FShaderParameterBindings const &,class FShaderParametersMetadata const *,void const *)" (__imp_?SetShaderParameters@@YAXAEAVFRHIComputeCommandList@@PEAVFRHIComputeShader@@AEBVFShaderParameterBindings@@PEBVFShaderParametersMetadata@@PEBX@Z) referenced in function "void __cdecl FComputeShaderUtils::Dispatch<class FSlimeCS>(class FRHIComputeCommandList &,class TShaderRefBase<class FSlimeCS,class FShaderMapPointerTable> const &,class FShaderParametersMetadata const *,class FSlimeCS::FParameters const &,struct UE::Math::TIntVector3<int>)" (??$Dispatch@VFSlimeCS@@@FComputeShaderUtils@@YAXAEAVFRHIComputeCommandList@@AEBV?$TShaderRefBase@VFSlimeCS@@VFShaderMapPointerTable@@@@PEBVFShaderParametersMetadata@@AEBVFParameters@FSlimeCS@@U?$TIntVector3@H@Math@UE@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl AddCopyTexturePass(class FRDGBuilder &,class FRDGTexture *,class FRDGTexture *,struct FRHICopyTextureInfo const &)" (__imp_?AddCopyTexturePass@@YAXAEAVFRDGBuilder@@PEAVFRDGTexture@@1AEBUFRHICopyTextureInfo@@@Z) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2001: unresolved external symbol "public: virtual __cdecl FShaderType::~FShaderType(void)" (??1FShaderType@@UEAA@XZ)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) enum ERHIFeatureLevel::Type GMaxRHIFeatureLevel" (__imp_?GMaxRHIFeatureLevel@@3W4Type@ERHIFeatureLevel@@A) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) struct FRHIGlobals GRHIGlobals" (__imp_?GRHIGlobals@@3UFRHIGlobals@@A) referenced in function "void __cdecl FComputeShaderUtils::Dispatch<class FSlimeCS>(class FRHIComputeCommandList &,class TShaderRefBase<class FSlimeCS,class FShaderMapPointerTable> const &,class FShaderParametersMetadata const *,class FSlimeCS::FParameters const &,struct UE::Math::TIntVector3<int>)" (??$Dispatch@VFSlimeCS@@@FComputeShaderUtils@@YAXAEAVFRHIComputeCommandList@@AEBV?$TShaderRefBase@VFSlimeCS@@VFShaderMapPointerTable@@@@PEBVFShaderParametersMetadata@@AEBVFParameters@FSlimeCS@@U?$TIntVector3@H@Math@UE@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) unsigned int GNumExplicitGPUsForRendering" (__imp_?GNumExplicitGPUsForRendering@@3IA) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: static struct FClearValueBinding const FClearValueBinding::White" (__imp_?White@FClearValueBinding@@2U1@B) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class UE::Trace::FChannel & RHICommandsChannel" (__imp_?RHICommandsChannel@@3AEAVFChannel@Trace@UE@@EA) referenced in function "public: virtual void __cdecl FRHICommand<struct FRHICommandSetShaderUnbinds<class FRHIComputeShader>,struct FRHICommandSetShaderUnbindsString1441>::ExecuteAndDestruct(class FRHICommandListBase &,struct FRHICommandListDebugContext &)" (?ExecuteAndDestruct@?$FRHICommand@U?$FRHICommandSetShaderUnbinds@VFRHIComputeShader@@@@UFRHICommandSetShaderUnbindsString1441@@@@UEAAXAEAVFRHICommandListBase@@AEAUFRHICommandListDebugContext@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class FRHICommandListExecutor GRHICommandList" (__imp_?GRHICommandList@@3VFRHICommandListExecutor@@A) referenced in function "void __cdecl FComputeShaderUtils::Dispatch<class FSlimeCS>(class FRHIComputeCommandList &,class TShaderRefBase<class FSlimeCS,class FShaderMapPointerTable> const &,class FShaderParametersMetadata const *,class FSlimeCS::FParameters const &,struct UE::Math::TIntVector3<int>)" (??$Dispatch@VFSlimeCS@@@FComputeShaderUtils@@YAXAEAVFRHIComputeCommandList@@AEBV?$TShaderRefBase@VFSlimeCS@@VFShaderMapPointerTable@@@@PEBVFShaderParametersMetadata@@AEBVFParameters@FSlimeCS@@U?$TIntVector3@H@Math@UE@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) bool GIsThreadedRendering" (__imp_?GIsThreadedRendering@@3_NA) referenced in function "public: class TRefCountPtr<class FGraphEvent> __cdecl TGraphTask<class TEnqueueUniqueRenderCommandType<struct `public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)'::`2'::SceneDrawCompletionName,class `public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)'::`2'::<lambda_1> > >::FConstructor::ConstructAndDispatchWhenReady<class `public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)'::`2'::<lambda_1> >(class `public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)'::`2'::<lambda_1> &&)" (??$ConstructAndDispatchWhenReady@V<lambda_1>@?1??DispatchGameThread@FSlimeCSInterface@@SAXUFSlimeCSDispatchParams@@@Z@@FConstructor@?$TGraphTask@V?$TEnqueueUniqueRenderCommandType@USceneDrawCompletionName@?1??DispatchGameThread@FSlimeCSInterface@@SAXUFSlimeCSDispatchParams@@@Z@V<lambda_1>@?1??23@SAX0@Z@@@@@QEAA?AV?$TRefCountPtr@VFGraphEvent@@@@$$QEAV<lambda_1>@?1??DispatchGameThread@FSlimeCSInterface@@SAXUFSlimeCSDispatchParams@@@Z@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class TAtomic<bool> GMainThreadBlockedOnRenderThread" (__imp_?GMainThreadBlockedOnRenderThread@@3V?$TAtomic@_N@@A) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)" (?DispatchGameThread@FSlimeCSInterface@@SAXUFSlimeCSDispatchParams@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class UE::Trace::FChannel & RenderCommandsChannel" (__imp_?RenderCommandsChannel@@3AEAVFChannel@Trace@UE@@EA) referenced in function "public: void __cdecl TEnqueueUniqueRenderCommandType<struct `public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)'::`2'::SceneDrawCompletionName,class `public: static void __cdecl FSlimeCSInterface::DispatchGameThread(struct FSlimeCSDispatchParams)'::`2'::<lambda_1> >::DoTask(enum ENamedThreads::Type,class TRefCountPtr<class FGraphEvent> const &)" (?DoTask@?$TEnqueueUniqueRenderCommandType@USceneDrawCompletionName@?1??DispatchGameThread@FSlimeCSInterface@@SAXUFSlimeCSDispatchParams@@@Z@V<lambda_1>@?1??23@SAX0@Z@@@QEAAXW4Type@ENamedThreads@@AEBV?$TRefCountPtr@VFGraphEvent@@@@@Z)
27>Module.ShadeUp.cpp.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) struct FCsvCategory _GCsvCategory_GPU" (__imp_?_GCsvCategory_GPU@@3UFCsvCategory@@A) referenced in function "public: static void __cdecl FSlimeCSInterface::DispatchRenderThread(class FRHICommandListImmediate &,struct FSlimeCSDispatchParams)" (?DispatchRenderThread@FSlimeCSInterface@@SAXAEAVFRHICommandListImmediate@@UFSlimeCSDispatchParams@@@Z)
27>C:\Users\chris\Documents\Unreal Projects\Test\Plugins\ShadeUp\Binaries\Win64\UnrealEditor-ShadeUp.dll : fatal error LNK1120: 58 unresolved externals
27>[6/10] Link [x64] UnrealEditor-ShadeUp.lib
27>   Creating library C:\Users\chris\Documents\Unreal Projects\Test\Plugins\ShadeUp\Intermediate\Build\Win64\x64\UnrealEditor\Development\ShadeUp\UnrealEditor-ShadeUp.lib and object C:\Users\chris\Documents\Unreal Projects\Test\Plugins\ShadeUp\Intermediate\Build\Win64\x64\UnrealEditor\Development\ShadeUp\UnrealEditor-ShadeUp.exp
27>[7/10] Resource Default.rc2
27>[8/10] Link [x64] UnrealEditor-Test.dll
27>   Creating library C:\Users\chris\Documents\Unreal Projects\Test\Intermediate\Build\Win64\x64\UnrealEditor\Development\Test\UnrealEditor-Test.sup.lib and object C:\Users\chris\Documents\Unreal Projects\Test\Intermediate\Build\Win64\x64\UnrealEditor\Development\Test\UnrealEditor-Test.sup.exp
27>[9/10] Link [x64] UnrealEditor-Test.lib
27>   Creating library C:\Users\chris\Documents\Unreal Projects\Test\Intermediate\Build\Win64\x64\UnrealEditor\Development\Test\UnrealEditor-Test.lib and object C:\Users\chris\Documents\Unreal Projects\Test\Intermediate\Build\Win64\x64\UnrealEditor\Development\Test\UnrealEditor-Test.exp
27>Total time in Parallel executor: 52.82 seconds
27>Total execution time: 68.77 seconds
27>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.MakeFile.Targets(50,5): error MSB3073: The command ""C:\Program Files\Epic Games\UE_5.3\Engine\Build\BatchFiles\Rebuild.bat" TestEditor Win64 Development -Project="C:\Users\chris\Documents\Unreal Projects\Test\Test.uproject" -WaitMutex -FromMsBuild" exited with code -1.
27>Done building project "Test.vcxproj" -- FAILED.
========== Rebuild All: 47 succeeded, 1 failed, 1 skipped ==========
========== Rebuild completed at 11:13 and took 01:30,766 minutes ==========
```

It is basically an empty Project, I am on Windows 11.

**Comment by ChrisSchroeder01** [comment]
I want to apologize, I overlooked some details on the documentation page: https://unreal.shadeup.dev/docs/cli

**Comment by AskingQuestions** [comment]
No worries, glad you were able to solve it on your own 🙂

This issue might help people searching in the future so for reference I suspect it was the module issue described below (feel free to correct me):

> [!WARNING]
> DO NOT use the default module provided with your plugin (it has the same name as your plugin). Modules containing shaders need a special loading phase configured, this loading phase is not included in the default module.

**Comment by ChrisSchroeder01** [comment]
Exactly! That was my issue 😅

**Comment by ChrisSchroeder01** [comment]
Ehm, I am encountering a few other "Problems"/Unclarities, I don't know if i should make for each one a Issue here on Github? I tried to join the discord via the Website to get help there but the Invite link is expired?

**Comment by AskingQuestions** [comment]
Oops, it did expire. It was just a link to the [Unreal Source](https://discord.gg/unrealsource) dc which I don't browse that much. Feel free to post a new issue with a list of the problems you're having and I'll do my best to help
